addons/project/static/src/js/project.js

```diff
@@ -110,4 +111,19 @@ KanbanRecord.include({
         }
     },
 });
+KanbanRenderer.include({
+    /**
+     * Overrides the default method to return custom columns for the project.task kanban view.
+     *
+     * @private
+     * @override
+    */
+    _getGhostColumns: function () {
+        if (this.state.model === 'project.task') {
+            return [_t('New'), _t('Assigned'), _t('In progress'), _t('Done')];
```

Comment:
In general, we try to avoid doing this as this is a bad pattern. I suggest to use the kanbanExamplesRegistry to encode this as well (in addition to the columns setups to display in the dialog).